libbeat/common/kubernetes/metadata.go

```diff
@@ -52,6 +52,15 @@ type MetaGeneratorConfig struct {
 
 type metaGenerator = MetaGeneratorConfig
 
+// DefaultMetaGeneratorConfig initializes and returns a new MetaGeneratorConfig with default values
+func DefaultMetaGeneratorConfig() MetaGeneratorConfig {
+	return MetaGeneratorConfig{
+		IncludeCreatorMetadata: true,
+		LabelsDedot:            true,
+		AnnotationsDedot:       true,
+	}
+}
```

Comment:
For consistency could you also use this `DefaultMetaGeneratorConfig()` to initialize the `metaGenerator` in `NewMetaGenerator` some lines below?